scripts/build.sh

```diff
@@ -506,6 +497,64 @@ find "../common/system/priv-app/" -maxdepth 1 -mindepth 1 -printf '%P\n' | xargs
 find "../common/system/priv-app/" -maxdepth 1 -mindepth 1 -printf '%P\n' | xargs -I placeholder sudo find "$SYSTEM_MNT/priv-app/placeholder" -exec setfattr -n security.selinux -v "u:object_r:system_file:s0" {} \; || abort
 echo -e "Add extra packages done\n"
 
+if [[ "$ROOT_SOL" == "kernelsu" ]]; then
+    echo "Copy KernelSU kernel"
+    mv "$WORK_DIR/wsa/$ARCH/Tools/kernel" "$WORK_DIR/wsa/$ARCH/Tools/kernel_origin"
+    cp "$WORK_DIR/kernelsu/kernel" "$WORK_DIR/wsa/$ARCH/Tools/kernel"
+    echo -e "Copy KernelSU kernel done\n"
+    echo "Add auto-install for KernelSU Manager"
+    # Copy APK
+    DAT_APP="$SYSTEM_MNT/data-app"
+    sudo mkdir "$DAT_APP"
+    sudo cp "$KERNELSU_APK_PATH" "$DAT_APP/"
+    sudo chmod 0755 "$DAT_APP/"
+    sudo chmod 0644 "$DAT_APP/KernelSU.apk"
+    sudo find "$DAT_APP/" -exec chown root:root {} \;
+    sudo find "$DAT_APP/" -exec setfattr -n security.selinux -v "u:object_r:system_file:s0" {} \; || abort
+    # Setup script
```

Comment:
@hmtheboy154 
Vietnamese: Bác có cao kiến gì không ạ. Em thử nãy giờ vẫn chưa làm được script auto-install cái KernelSU
English: Do you have any idea ?